### PR TITLE
Export react tabs context

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Changes
- Updated react-tabs TabList context to follow same pattern as other components
- Exported context from component index
- Exported react-tabs context from react-components

This allows recomposition as the renderTabList method requires context values.

### CODE REVIEW

File: change/@fluentui-react-components-817ab317-7384-4c3e-acaf-06eb10fa5f76.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added context exports for react-tabs",
```
Comment (khmakoto) [inline change/@fluentui-react-components-817ab317-7384-4c3e-acaf-06eb10fa5f76.json:3]:
```suggestion
  "comment": "chore: Added context exports for react-tabs",
```

File: change/@fluentui-react-tabs-0f215c5f-aaf8-4c1d-9283-40dac84261c3.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added context exports",
```
Comment (khmakoto) [inline change/@fluentui-react-tabs-0f215c5f-aaf8-4c1d-9283-40dac84261c3.json:3]:
```suggestion
  "comment": "chore: Added context exports",
```

**Review by khmakoto — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              | `60.588 kB`<br />`16.969 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `188.355 kB`<br />`52.98 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            | `34.751 kB`<br />`11.453 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |   `6.069 kB`<br />`2.053 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/fc896528cd40ac5e32c0fb125fb00c502a471cab'>fc896528cd40ac5e32c0fb125fb00c502a471cab</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/Avatar-mount_base.html">1630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/Avatar-mount.html">1717</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/Button-mount_base.html">1170</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/Button-mount.html">1154</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/FluentProvider-mount_base.html">2030</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/FluentProvider-mount.html">2039</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/FluentProviderWithTheme-mount_base.html">804</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/FluentProviderWithTheme-mount.html">777</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">730</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">719</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">773</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">794</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/InfoButton-mount_base.html">692</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/InfoButton-mount.html">709</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/MakeStyles-mount_base.html">2401</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/MakeStyles-mount.html">2354</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/Persona-mount_base.html">3760</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/Persona-mount.html">3700</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/SpinButton-mount_base.html">3204</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26552/perf-test-react-components/SpinButton-mount.html">3148</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Pull request merged**